### PR TITLE
Drop scan report

### PR DESCRIPTION
**Pull request by manelmontilla**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ RUN apk add postgresql-client
 # add flyway
 RUN apk add --no-cache --update openjdk8-jre-base bash gettext libc6-compat
 
-ARG FLYWAY_VERSION=9.19.3
```
Comment (jesusfcr) [inline Dockerfile:26]: Why the downgrade of flyway?

**Review by jesusfcr — CHANGES_REQUESTED** [review]
Missing `go mod tidy`

File: Dockerfile (hunk 1 above)
Comment (manelmontilla) [reply Dockerfile:26]: It was a mistake, good catch reverting the line to the previous version:.

**Review by jesusfcr — APPROVED** [review]
LGTM

**Pull request merged**